README.lt.md

```
@@ -0,0 +1,502 @@
+
+- **Balso atpažinimas prastas/lėtas.**
+→ Naudok debesies STT. Ant Intel naudok tik našumo branduolius.
+
+- **Kaip tvarkomi duomenys?**
+→ Balsas ir pokalbio turinys saugomi lokaliai ir nesiunčiami į serverius.
+
+### [📥 Atsisiųsti](https://github.com/kapitalismho/PuriPuly-heart/releases/latest)
+
+---
+
+## Vertimo palyginimas
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)